client/workflows/demos/registry/sklearn-census-end-to-end.ipynb

```diff
@@ -0,0 +1,383 @@
+    "    run_experiment(hyperparams)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_run = expt.expt_runs.sort(\"metrics.val_acc\", descending=True)[0]\n",
+    "print(\"Validation Accuracy: {:.4f}\".format(best_run.get_metric(\"val_acc\")))\n",
+    "\n",
+    "best_hyperparams = best_run.get_hyperparameters()\n",
+    "print(\"Hyperparameters: {}\".format(best_hyperparams))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = linear_model.LogisticRegression(multi_class='auto', **best_hyperparams)\n",
+    "model.fit(X_train, y_train)\n",
+    "train_acc = model.score(X_train, y_train)\n",
+    "print(\"Training accuracy: {:.4f}\".format(train_acc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Register Model for deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "registered_model = client.get_or_create_registered_model(\n",
+    "    name=\"census-sklearn\", labels=[\"tabular\", \"sklearn\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.1 Register from the model object\n",
+    "#### If you are in the same file where you have the model object handy, use the code below to package the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from verta.environment import Python\n",
+    "\n",
+    "model_version_v1 = registered_model.create_standard_model_from_sklearn(\n",
+    "    model,\n",
+    "    environment=Python(requirements=[\"scikit-learn\"]),\n",
+    "    name=\"v1\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2 (OR) Register a serialized version of the model using the VertaModelBase"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from verta.registry import VertaModelBase\n",
+    "\n",
+    "class CensusIncomeClassifier(VertaModelBase):\n",
+    "    def __init__(self, artifacts):\n",
+    "        self.model = cloudpickle.load(artifacts[\"serialized_model\"])\n",
+    "        \n",
+    "    def predict(self, batch_input):\n",
+    "        results = []\n",
+    "        for one_input in batch_input:\n",
+    "            sself.model.predict(one_input)\n",
+    "        return results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_version_v2 = registered_model.create_standard_model(\n",
+    "    model_cls=CensusIncomeClassifier,\n",
+    "    environment=Python(requirements=[\"scikit-learn\"]),\n",
+    "    artifacts={\"model.pkl\" : \"moodel.pkl\"},\n",
```

Comment:
I think we're missing a
```python
with open("model.pkl", "rb") as f:
    cloudpickle.dump(model, f)
```
somewhere. Or, this line could be
```suggestion
    "    artifacts={\"model.pkl\" : model},\n",
```

Also, "moodel.pkl" might be a typo 🐄